<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-06T04:52:37Z" name="Suite" finished-at="2022-01-06T04:53:29Z" duration-ms="52238">
    <groups>
    </groups>
    <test started-at="2022-01-06T04:52:37Z" name="Test" finished-at="2022-01-06T04:53:29Z" duration-ms="52238">
      <class name="tests.LoginTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:52:37Z" name="beforeMethod" finished-at="2022-01-06T04:52:46Z" duration-ms="9309" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="loginWithWrongEmailPasswordTest()[pri:1, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:52:46Z" name="loginWithWrongEmailPasswordTest" finished-at="2022-01-06T04:52:49Z" duration-ms="3032" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithWrongEmailPasswordTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:52:49Z" name="afterMethod" finished-at="2022-01-06T04:52:50Z" duration-ms="1214" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:52:50Z" name="beforeMethod" finished-at="2022-01-06T04:52:59Z" duration-ms="8991" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="loginWithEmptyEmailTest()[pri:2, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:52:59Z" name="loginWithEmptyEmailTest" finished-at="2022-01-06T04:53:02Z" duration-ms="2692" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithEmptyEmailTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:53:02Z" name="afterMethod" finished-at="2022-01-06T04:53:03Z" duration-ms="1422" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:53:03Z" name="beforeMethod" finished-at="2022-01-06T04:53:12Z" duration-ms="8448" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="loginWithEmptyPasswordTest()[pri:3, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:53:12Z" name="loginWithEmptyPasswordTest" finished-at="2022-01-06T04:53:14Z" duration-ms="2702" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithEmptyPasswordTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:53:14Z" name="afterMethod" finished-at="2022-01-06T04:53:16Z" duration-ms="1124" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:53:16Z" name="beforeMethod" finished-at="2022-01-06T04:53:25Z" duration-ms="9271" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="loginWithinvalidEmailTest()[pri:4, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:53:25Z" name="loginWithinvalidEmailTest" finished-at="2022-01-06T04:53:28Z" duration-ms="2865" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter a username or card number without special character.] but found [Please enter a username or card number without special characters.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter a username or card number without special character.] but found [Please enter a username or card number without special characters.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at tests.LoginTest.loginWithinvalidEmailTest(LoginTest.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithinvalidEmailTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.LoginTest@5e17553a]" started-at="2022-01-06T04:53:28Z" name="afterMethod" finished-at="2022-01-06T04:53:29Z" duration-ms="1148" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- tests.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
